BnbnavNetClient/Models/Player.cs

```diff
@@ -1,7 +1,24 @@
-﻿namespace BnbnavNetClient.Models;
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Avalonia;
+using Avalonia.Animation;
+using Avalonia.Media;
+using BnbnavNetClient.Helpers;
+using BnbnavNetClient.Services;
+using Timer = System.Timers.Timer;
+
+namespace BnbnavNetClient.Models;
```

Comment:
No new line after namespace, no explicit private

Reply:
Please elaborate

Reply:
Seems like our project style has an extra line after each namespace declaration - if that is to be changed it should be in a separate PR